content/posts/bayestestR_v2.Rmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Become a Bayesian master with bayestestR v2"
+author: "easystats"
+date: 2019-05-30
+categories: ["R", "bayestestR"]
+tags: ["R", "easystats", "bayestestR"]
+---
+
+
+```{r message=FALSE, warning=FALSE, include=FALSE}
+library(knitr)
+options(knitr.kable.NA = '',
+        digits=2,
+        dpi=450)
+```
+
+
+```{r message=FALSE, warning=FALSE, echo=FALSE, out.width = "200px"}
+include_graphics("https://github.com/easystats/bayestestR/raw/master/man/figures/logo.png")
+```
+
+
+
+## bayestestR v2 is here!
+
+As you might have heard from [previous posts](https://easystats.github.io/blog/posts/), [we](https://github.com/orgs/easystats/people) have recently started to collaborate around the new [**easystats project**](https://github.com/easystats), a set of packages designed to make your life easier. One of the package, [**bayestestR**](https://github.com/easystats/bayestestR), has just been updated on CRAN.
```

Comment:
I would maybe add: "...around the new easystats project, which is a suite of packages..." Just a few word changes/ideas

Reply:
Also, in the next sentence, package should be plural: "One of the packageS..."